pkg/cluster/internal/create/create.go

```diff
@@ -57,6 +63,11 @@ func Cluster(ctx *context.Context, cfg *config.Cluster, opts *Options) error {
 	// may be constructed in memory rather than from disk)
 	encoding.Scheme.Default(cfg)
 
+	// warn if cluster name might typically be too long
+	if len(ctx.Name()) > clusterNameMax {
+		log.Warnf("cluster name %q is too long, might not work properly on some systems", ctx.Name())
```

Comment:
nit: s/is/is probably/
we make no promises :sweat_smile: 
maybe also s/might/this might/

Reply:
done